janitor/functions.py

```diff
@@ -3140,17 +3140,63 @@ def update_where(
         # 4 8  0
 
     :param df: The pandas DataFrame object.
-    :param conditions: conditions used to update a target column and target
-        value
+    :param conditions: Conditions used to update a target column
+                       and target value.
```

Comment:
Indentation needs to be fixed here. I noticed this happened in your previous PRs. Do you have a code editor setting that might be interfering with this?

```suggestion
        and target value.
```

Also, can you fix the others? I'll mark them with a copy/pasted note.